tests/platform_tests/test_reload_dpu.py

```diff
@@ -0,0 +1,79 @@
+"""
+Tests for the `reboot and reload ...` commands in DPU 
+"""
+
+import logging
+import pytest
+import time
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts     
+from tests.common.platform.interface_utils import check_interface_status_of_up_ports
+from tests.common.utilities import wait_until
+from tests.common.platform.processes_utils import wait_critical_processes
+from tests.common.reboot import reboot,wait_for_startup,REBOOT_TYPE_COLD
+from tests.common.config_reload import config_force_option_supported, config_system_checks_passed
+from tests.common.platform.device_utils_dpu import *
+from tests.common.helpers.platform_api import chassis, module
+from tests.platform_tests.api.conftest import platform_api_conn
+
+pytestmark = [
+    pytest.mark.topology('t1')
+]
+
+
+def test_dpu_ping_after_reboot(duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
+    """
+    @summary: Verify output of `config chassis modules startup <DPU_Number>`
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    ip_address_list = []
+    dpu_powered_on_list = []
+    num_modules = int(chassis.get_num_modules(platform_api_conn))
+    output_dpu_status = duthost.show_and_parse('show chassis module status')
+
+    for index in range(len(output_dpu_status)):
+        parse_output = output_dpu_status[index]
+        if parse_output['oper-status'] != 'Offline':
```

Comment:
Why some of the DPUs are expected to be in “Offline” status?

Reply:
Changed it. It was added before creating the power on fixture, hence the confusion.